integration_tests/dbt_project.yml

```diff
@@ -10,11 +10,12 @@ clean-targets:         # directories to be removed by `dbt clean`
 
 vars:
   ## Uncomment when regenerating docs
-  # shopify_using_fulfillment_event: true
-  # shopify_using_all_metafields: true
-  # shopify__standardized_billing_model_enabled: true
-
+  shopify_using_fulfillment_event: true
+  shopify_using_all_metafields: true
+  shopify__standardized_billing_model_enabled: true
+  shopify_using_abandoned_checkout: true
```

Comment:
Is this necessary? Isn't it enabled by default already?

Reply:
Ah yes it is, removed